torch/_subclasses/fake_tensor.py

```diff
@@ -70,22 +108,25 @@ def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
             out_device = new_kwargs.pop("device", new_kwargs["input"].device)
             with no_dispatch():
                 input = new_kwargs.pop("input").to("meta")
-                return FakeTensor(torch.ops.aten._to_copy(input, **new_kwargs), out_device)
+                return FakeTensor(
+                    torch.ops.aten._to_copy(input, **new_kwargs), out_device
+                )
 
-        r = super().__torch_dispatch__(func, types, args, kwargs)
+        if _is_tensor_constructor(func):
+            assert not func in _non_kwarg_device_constructors
+            _, new_kwargs = normalize_function(
+                func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+            )
+            # cpu is default device if none is specified
+            out_device = new_kwargs.pop("device", torch.device("cpu"))
```

Comment:
technically it's `torch.get_default_tensor_type()` lol but ok

Reply:
which is immutable 😛